### ISSUE TITLE
Create timeline view of quest completions

### ISSUE THREAD

**Issue by thaddeus** [TarkovTracker/TarkovTracker#15]
Add a view that shows quest completions of a generic progress object so that you could view the history of your current or past progress.

**Comment by Hanzik** [comment]
Could you elaborate what this should look like? Is this something like an interactive graph of quest chains like this one?

![image](https://user-images.githubusercontent.com/5736517/126044861-9efdcc7b-c458-4dd6-8da8-2534d87f4c39.png)

Except this one would highlight quests completed, quests in progress etc.?

**Comment by thaddeus** [comment]
The chain view is something that looks cool, but I've never seen an interactive version I thought looked nice, in my personal opinion. 

My original thought for this, was more along the lines of a line chart, starting when your first quest was completed, progressing to today, and displaying quests completed per day or week depending upon a toggle. https://gionkunz.github.io/chartist-js/examples.html - a stretch achievement for this would be being able to select a day/week and show a list of quests marked complete in that timeframe below the graph.

**Comment by Hanzik** [comment]
Yeah, it's, at this point, a lot of quests in the mix, which makes making everything look nice and clear pretty difficult. 

But I do understand your idea, looks quite alright, I will give it some thought.

**Comment by umpulumpa** [comment]
I made this for a discord bot I made using the Api, is this what you mean?
![graph](https://user-images.githubusercontent.com/58427226/169717118-b6a88bf1-1c7d-453f-848d-db1821ba8ae7.PNG)
![graph 2](https://user-images.githubusercontent.com/58427226/169717151-605155f4-c96d-4bb0-a495-9f1c969fec1b.PNG)

**Comment by thaddeus** [comment]
@umpulumpa Yeah. That's generically what I'm going for. Would love for people to be able to see that on the overview page to see how they're progressing (and eventually see how they're progressing compared to previous wipes)
